graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/DgsWebMvcAutoConfiguration.kt

```diff
@@ -60,7 +61,9 @@ open class DgsWebMvcAutoConfiguration {
     @Qualifier("dgsObjectMapper")
     @ConditionalOnMissingBean(name = ["dgsObjectMapper"])
     open fun dgsObjectMapper(): ObjectMapper {
-        return jacksonObjectMapper().registerModule(JavaTimeModule())
+        return Jackson2ObjectMapperBuilder.json()
```

Comment:
So it seems to install kotlin modules, but not JavaTimeModule, correct? https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/converter/json/Jackson2ObjectMapperBuilder.html

Reply:
It does install `JavaTimeModule`, see [here](https://github.com/spring-projects/spring-framework/blob/6bd7f0231dcc47e8099000dea301157788fd5b15/spring-web/src/main/java/org/springframework/http/converter/json/Jackson2ObjectMapperBuilder.java#L880). I think the Javadoc is just out of date.

Reply:
Oh, the Javadoc refers to `jackson-datatype-jsr310`, which is the project the time module used to live in.